contracts/native-interchain-token.clar

```diff
@@ -366,18 +366,13 @@
 ;; ######################
 ;; ######################
 
-(define-constant DEPLOYER tx-sender)
-
-(define-data-var token-type (optional uint) none)
-
 (define-data-var is-started bool false)
 (define-read-only (get-is-started)
     (ok (var-get is-started))
 )
 
 (define-public (setup
         (token-id_ (buff 32))
-        (token-type_ uint)
```

Comment:
I suggest to leave the `token-type` and allow it to be either:
`TOKEN-TYPE-NATIVE-INTERCHAIN-TOKEN = 0`
or
`TOKEN-TYPE-MINT-BURN = 4`.

Currently ITS only supports `TOKEN-TYPE-NATIVE-INTERCHAIN-TOKEN`, but in the future after token linking is implemented it will also support `TOKEN-TYPE-MINT-BURN`. This way the NIT contract code can be forward compatible with what will be implemented next.

https://github.com/axelarnetwork/interchain-token-service/blob/main/contracts/interfaces/ITokenManagerType.sol#L15

Reply:
got it